frontend/public/components/search.tsx

```diff
@@ -200,6 +202,31 @@ const SearchPage_: React.FC<SearchProps & StateProps & DispatchProps> = (props)
     );
   };
 
+  const getCreateLinkButton = (kind: string) => {
```

Comment:
This needs a check on canCreate().  I think this is also why the testcase is failing.